client/src/App.js

```diff
@@ -30,10 +31,23 @@ function App() {
   const [hidden, setHidden] = React.useState(false);
   const [innerHidden, setInnerHidden] = React.useState(false);
   const [isFullScreen, setIsFullScreen] = React.useState(false);
+  const [userData, setUserData] = React.useState(null);
   const toLightBox = () => {
     document.querySelector(".lightbox").scrollIntoView({ behavior: "smooth" });
   };
 
+  React.useEffect(() => {
+    axios({
+      method: "GET",
+      withCredentials: true,
+      url: "http://localhost:3001/user",
+    }).then((res) => {
+      setUserData(res.data);
+      //console.log(res.data); //brings side effects of affecting browser's performance -- infinity error
+      //console.log(userData.username) //brings side effects of affecting browser's performance -- infinity error
```

Comment:
We can just remove these `console.log`s since we'll add a little test piece of html further down to see the username value get spit out
```suggestion
```